### DISCUSSION TITLE
MIDI 2.0

### DISCUSSION THREAD

**Discussion by Klaviernatum** [shanteacontrols/OpenDeck#82] [Q&A] [answered]
It's OpenDeck MIDI 2.0-Ready?

**Comment by paradajz** [comment, marked as answer]
No

**Reply by Klaviernatum** [reply]
It's in your current agenda the closer implementation of it? Also want to know if actually its MPE compatible?

**Reply by paradajz** [reply]
No and no. I haven't seen real-world use of MIDI 2.0 yet so it's not something I'm willing to spend time on in the foreseeable future. I also don't know much about MPE so I can't give you clear answer - OpenDeck is certainly not officially MPE compatible but again, I don't know the spec.
